packages/rabbitmq/src/lib/amqp/connection.ts

```diff
@@ -259,17 +259,12 @@ export class AmqpConnection {
       first()
     );
 
-    await this.publish(
-      requestOptions.exchange,
-      requestOptions.routingKey,
-      payload,
-      {
-        replyTo: DIRECT_REPLY_QUEUE,
-        correlationId,
-        headers: requestOptions.headers,
-        expiration: requestOptions.expiration,
-      }
-    );
+    this.publish(requestOptions.exchange, requestOptions.routingKey, payload, {
```

Comment:
Are you skipping the git hooks locally?  It seems like this is reverting Prettier's formatting.

Also, why is the `await` being dropped here and on line 389 below?

Reply:
I'm not skipping the hooks, it seems that this is the "new way to go". I've added a `lint` CI job which is passing, so I don't think there is an issue here.

If an `await` was removed, it's surely because the called function was not async, otherwise, it would have thrown a lint error in either case :)